### PR TITLE
Update dLibra translator.

### PR DESCRIPTION
**Pull request by kgronowski**
Added support for dLibra 6.

### CODE REVIEW

**Comment by AbeJellinek** [comment]
Hey, thanks for contributing this! Sorry I'm just getting to it now.

As the site's HTML says:

```html
<!--type is "document" for import all tags into zotero-->
```

But it's kind of a cop-out to import every item as `document`, and I think we can do better :) It looks like the dLibra 5 translator used the RDF - why aren't we doing that with dLibra 6? [The RDF they give us](https://demo.dl.psnc.pl/dlibra/rdf.xml?type=e&id=1243) has the correct item type. It's in Polish, so the translator will have to have a Polish -> Zotero item type mapper, but that's not a big deal.

File: dLibra.js [hunk 1]
```diff
@@ -1,135 +1,228 @@
...
+		translator.setHandler("itemDone", function (obj, item) {
+			if (item.extra) item.notes.push(item.extra);
+			item.extra = "";
+			item.itemID = "";
+			item.complete();
 		});
-		
-	})
+		translator.getTranslatorObject(function (trans) {
+			trans.defaultUnknownType = 'document';
```
Comment (AbeJellinek) [inline dLibra.js:117]: map item types instead of doing this

**Review by AbeJellinek — CHANGES_REQUESTED** [review]

**Comment by AbeJellinek** [comment]
Trying to test but the demo site is returning an internal server error. The first site in the tests works but looks to be dLibra 5.

**Comment by kgronowski** [comment]
Thanks for the notice. dLibra 6 demo should be working fine now.

File: dLibra.js [hunk 2]
```diff
@@ -1,135 +1,282 @@
...
+					}
+				],
+				"notes": [],
+				"seeAlso": [],
+				"url": "http://cyfrowa.bibliotekakolbuszowa.pl/dlibra/docmetadata?id=945"
+			}
+		]
+	},
+	{
+		"type": "web",
```
Comment (AbeJellinek) [inline dLibra.js:223]: These three tests for multiples aren't working.

**Review by AbeJellinek — CHANGES_REQUESTED** [review]
Thanks! A few more comments.

**Comment by AbeJellinek** [comment]
Thanks so much! Linted and merged.

**Pull request merged**